metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py

```diff
@@ -1205,6 +1297,10 @@ def get_workunits_internal(self) -> Iterable[MetadataWorkUnit]:
             )
 
         self.reporter.report_stage_start("explore_metadata")
```

Comment:
please add a new `report_stage_start` call for this

Reply:
done